### PR TITLE
List addition test for add_assign

### PR DESCRIPTION
**Pull request by WillemWybo**
Just a small extra test to check add_assign with lists of compartments

### CODE REVIEW

File: testsuite/pytests/test_compartmental_model.py [hunk 1]
```diff
@@ -942,6 +942,16 @@ def test_add_assign(self):
         self.assertEqual(compartments[1]["comp_idx"], 1)
         self.assertEqual(compartments[1]["parent_idx"], 0)
 
+        cm2 = nest.Create('cm_default')
+        cm2.compartments = {"parent_idx": -1, "params": SP}
+        cm2.compartments += [{"parent_idx": 0, "params": DP[0]}, {"parent_idx": 0, "params": DP[0]}]
```
Comment (hakonsbm) [inline testsuite/pytests/test_compartmental_model.py:947]: Should the second `parent_idx` be 1 here? The test seems to pass regardless though.
```suggestion
        cm2.compartments += [{"parent_idx": 0, "params": DP[0]}, {"parent_idx": 1, "params": DP[0]}]
```

File: testsuite/pytests/test_compartmental_model.py [hunk 2]
```diff
@@ -942,6 +942,16 @@ def test_add_assign(self):
...
+        self.assertEqual(len(list(cm2.compartments)), 3)
+
+        cm3 = nest.Create('cm_default')
+        cm3.compartments = [{"parent_idx": -1, "params": SP}, {"parent_idx": 0, "params": DP[0]}]
+        cm3.compartments += {"parent_idx": 1, "params": DP[0]}
+        self.assertEqual(len(list(cm2.compartments)), 3)
```
Comment (hakonsbm) [inline testsuite/pytests/test_compartmental_model.py]: Just a small typo.

```suggestion
        self.assertEqual(len(list(cm3.compartments)), 3)
```

File: testsuite/pytests/test_compartmental_model.py (hunk 1 above)
Comment (WillemWybo) [reply testsuite/pytests/test_compartmental_model.py:947]: It's just a different (valid) tree structure. You can put whichever one you prefer.

File: testsuite/pytests/test_compartmental_model.py (hunk 2 above)
Comment (WillemWybo) [reply testsuite/pytests/test_compartmental_model.py]: Right, thanks for pointing out. Fixed it.

**Pull request merged**